clang/test/Driver/aix-ld.c

```diff
@@ -1120,3 +1120,18 @@
 // RUN:        -c \
 // RUN:   | FileCheck --check-prefixes=CHECK-K-UNUSED %s
 // CHECK-K-UNUSED: clang: warning: -K: 'linker' input unused [-Wunused-command-line-argument]
+
+// Check No Sanitizer on 32-bit AIX
+// This test verifies that the linker doesn't include '-latomic' when no sanitizers are enabled 
+// RUN: %clang -target powerpc-ibm-aix -m32 %s -### 2>&1 \
+// RUN: | FileCheck -check-prefix=CHECK-LD32-NO-SANITIZER %s 
+// CHECK-LD32-NO-SANITIZER-NOT: "-latomic"
+
+// Check enable AddressSanitizer on 32-bit AIX
+// FIXME: Running this test on non-AIX hosts will result in the following error:
+// LLVM ERROR: Sanitizer interface functions must be exported by export files on AIX
+// RUN: %if target={{.*aix.*}} %{ \
```

Comment:
Where is this diagnostic `Sanitizer interface functions must be exported by export files on AIX` defined and why is there a host difference? I am concerned this would cause friction for people improve the generic interface to inadvertently cause regression on AIX, since most contributors don't have access to AIX.

Reply:
> Where is this diagnostic `Sanitizer interface functions must be exported by export files on AIX` defined and why is there a host difference?

It is strictly in IBM's downstream at this time. It will likely need revisiting before upstreaming. I have requested the removal of the driver part from this PR.